### PR TITLE
jobs: BRD: EW changes

### LINKED ISSUE #3827: Bard's DOT Timers and Alerts Are 15 Seconds Off
## Description

Bard's two DOTs (Venomous Bite and Windbite including the upgraded versions) are 15 seconds off. This offset puts both the timers and warning off.

## Additional information

With Endwalker's release, Bard's DOTs are now 45 seconds in length.

### PR DESCRIPTION
**Pull request by Echoring**
Fixes #3827 , DoT time 45s now
Straight Shot Proc time 30s now
Fix broken song box due to change of #3545
Refactor repertoire timer, for in EW it no longer rely on DoT.
Also change code order to makes box order same as before #3818

### CODE REVIEW

File: ui/jobs/components/brd.ts [hunk 1]
```diff
@@ -57,20 +50,47 @@ export class BRDComponent extends BaseComponent {
...
+  override onUseAbility(id: string): void {
+    switch (id) {
+      case kAbility.MagesBallad:
+      case kAbility.ArmysPaeon:
+      case kAbility.theWanderersMinuet:
+        // Seem EW BRD's repertoire always tick every 3s after song start
+        // 45s and 0s not included
+        // FIXME: stop timer when song is at last 3s or ended
+        if (!this.is5x)
+          this.repertoireTimer.duration = 3;
```
Comment (quisquous) [inline ui/jobs/components/brd.ts:91]: Is it every 3 seconds, or is this the same as `kDoTTickInterval`? Is it tied to the start of the song, or is this like the mp ticker where it's based on server ticks?

Comment (Echoring) [reply ui/jobs/components/brd.ts:91]: Every 3 seconds (at least I can't tell any mismatch in 45s)
Tied to the start of the song, Tick on 42 39 36 33 30 27 24 21 18 15 12 9 6 3

Comment (quisquous) [reply ui/jobs/components/brd.ts:91]: Interesting!!

Comment (JLGarber) [reply ui/jobs/components/brd.ts:91]: Square normalized it heavily, for better or for worse, and we do indeed not get any benefit for running the song to zero.

**Pull request merged**